students/N0vA/lesson_05/except_exercise.py

```diff
@@ -20,10 +20,10 @@
     joke = fun(first_try[0])
 except NameError:
     print('There is no joke for topic - spam')
-try:
-    joke = fun(first_try[1])
-except NameError:
-    print('There is no joke for topic - cheese')
+    try:
+        joke = fun(first_try[1])
+    except NameError:
+        print('There is no joke for topic - cheese')
```

Comment:
this would be considered a separate try/except block.

This should look like:
```
try:
    joke = fun(first_try[0])
except NameError:
    joke = fun(first_try[1])
```